lesson18/script.js

```diff
@@ -0,0 +1,186 @@
+const slider = document.getElementById("js-slider");
+
+const createAttributedElements = ({tag,attrObj,str}) =>{
+    const element = document.createElement(tag);
+    Object.keys(attrObj).forEach((attribute) => {
+        element.setAttribute(attribute, attrObj[attribute]);
+    });
+    if(str !== undefined) element.textContent = str;
+    return element;
+}
```

Comment:
This function is a function to make Element, attribute, and text together. 
In Issue 17,Monaka-san gave me some advice.
Thank you so much!
https://github.com/nakagawamai/js-lesson/pull/19#discussion_r863614660